src/lib/utils.js

```diff
@@ -25,11 +26,12 @@ export async function readLineAsync(query) {
 export async function retryUntilSuccess(callbackFn) {
   try {
     return await callbackFn();
-  } catch {
-    return await retryUntilSuccess(callbackFn);
+  } catch (error) {
+    OutputView.printErrorMessage(error);
+    return retryUntilSuccess(callbackFn);
   }
```

Comment:
버그는 고쳐졌는데, util 함수가 View를 알게 되었네요!
아래처럼 params를 추가로 받아서 수정하거나, 함수를 view 폴더 안으로 옮겨야할 것 같아요~!

```suggestion
  } catch (error) {
    onError(error);
    return retryUntilSuccess(callbackFn);
  }
```

Reply:
그런 생각을 못했네요,,!!

1. 8578cfe783b2a27496c7ce23539dfb4c478a9d27 : 콜백 패턴을 활용해 util에서 InputView로 출력 로직을 분리했습니다. 하지만 이제는 InputView가 OutputView를 알게되었습니다.
2. 430375491a68f16fd0cb4cdb3406e9b936e11554 : InputView에서 Controller로 로직을 분리했습니다. Controller에서 유틸 함수와 InputView, OutputView 모두 제어합니다.

덕분에 Controller에서 전부 제어하는 로직을 모두 볼 수게 되었고, 뿐만 아니라 유틸과 뷰 간의 역할과 책임이 명확해지고, 결합도가 낮아진 듯해요 !!